### PR TITLE
Fix Shantotto weaponskill trade cs

### PR DESCRIPTION
**Pull request by kaincenteno**
When playing with the fiance, he noticed that there was something missing on his retribution weaponskill quest trade chat with Shantotto.

Shantotto CS 448 needs to also pass nil nil keyitem, otherwise message will be blank.

the problematic cs:
broken

after making changes:
fixed

### CODE REVIEW

File: scripts/zones/Windurst_Walls/npcs/Shantotto.lua [hunk 1]
```diff
@@ -19,7 +19,11 @@ function onTrade(player,npc,trade)
     local count = trade:getItemCount()
 
     if wsQuestEvent ~= nil then
-        player:startEvent(wsQuestEvent)
+        if wsQuestEvent == 448 then
+            player:startEvent(wsQuestEvent, nil, nil, 345)
```
Comment (ibm2431) [inline scripts/zones/Windurst_Walls/npcs/Shantotto.lua]: `dsp.keyItem.ANNALS_OF_TRUTH`

Comment (kaincenteno) [reply scripts/zones/Windurst_Walls/npcs/Shantotto.lua]: @ibm2431  changed, thanks for the review 😸

**Comment by TeoTwawki** [comment]
Don't close the pr next time. You can force push if you need to overwrite the branch and the pull request gets updated automatically.

**Comment by kaincenteno** [comment]
@TeoTwawki thank you will do a force push next time if this needs to be done.

**Pull request closed without merging**